packages/ui/src/external/RouteVisualisation/RouteVisualization.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext, useEffect, useLayoutEffect } from 'react';
+import { EntitiesContext, EntitiesProvider, RuntimeProvider, VisibleFlowsProvider } from '../../providers';
+import { Visualization } from '../../components/Visualization/Visualization';
+import { EventNotifier } from '../../utils';
+
+const Viz: React.FC<{ catalogUrl: string }> = ({ catalogUrl }) => {
+  const entitiesContext = useContext(EntitiesContext);
+  const visualEntities = entitiesContext?.visualEntities ?? [];
+  return (
+    <RuntimeProvider catalogUrl={catalogUrl}>
+      <VisibleFlowsProvider>
+        <Visualization className={'canvas-page'} entities={visualEntities} />
+      </VisibleFlowsProvider>
+    </RuntimeProvider>
+  );
+};
+
+export const RouteVisualization: React.FC<{ catalogUrl: string; code: string; codeChange: (code: string) => void }> = ({
```

Comment:
[curious-question] why the split component? :smile:

Reply:
For reusing EntityProvider and RuntimeProvider. If there is an another way, I'm happy to refactor :)